pkg/endpoint/endpoint.go

```diff
@@ -884,8 +887,9 @@ func (e *Endpoint) GetModel() *models.Endpoint {
 	if e == nil {
 		return nil
 	}
-	e.Mutex.RLock()
-	defer e.Mutex.RUnlock()
+	// NOTE: Using rlock on mutex directly because GetModelRLocked handles endpoint in disco state properly
```

Comment:
spelling

Reply:
what is wrong with spelling here?

Reply:
I'm all for partying but `disco` is not a state ;-)